src/components/POLine/hooks/useFundsWithRestrictedLocationsById/useFundsWithRestrictedLocationsById.js

```diff
@@ -0,0 +1,53 @@
+import {
+  useCallback,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
+
+import { useFundsById } from '../useFundsById';
+import { useGetLocationsByHoldingIds } from '../useGetLocationsByHoldingIds';
+
+export const useFundsWithRestrictedLocationsById = ({
```

Comment:
based on the name I should get a list of funds, not a boolean value of the limitation. Please match the name of the hook with the expected result